packages/webapp/src/containers/Animals/types.ts

```diff
@@ -28,3 +28,13 @@ export enum AnimalOrBatchKeys {
   ANIMAL = 'ANIMAL',
   BATCH = 'BATCH',
 }
+
+export enum AnimalSexes {
```

Comment:
This enum is not used. But the form container might want to use it when it sets up its own `isMaleSelected`, is that correct?